tests/test_frequency_weighting.py

```diff
@@ -8,8 +8,7 @@
 
 @pytest.mark.parametrize("weighting", ["A", "C"])
 @pytest.mark.parametrize("num_fractions", [1, 3])
-@pytest.mark.parametrize("freq_range", [(20, 20000), (0, 100),
-                                        (4000, 100_000)])
+@pytest.mark.parametrize("freq_range", [(20, 20000)])
```

Comment:
Splitted these into two tests. One that tests a frequency range where no warnings are emitted and one that tests the warnings